checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakVisitor.java

```diff
@@ -33,6 +34,9 @@
  */
 public class ResourceLeakVisitor extends CalledMethodsVisitor {
 
+  /** True if errors related to static owning fields should be suppressed. */
+  boolean permitStaticOwning;
```

Comment:
When I saw that the similarly-named field in `MustCallConsistencyAnalyzer` was package-private, I assumed that the reason was that this class and that one would share a single field value. That doesn't appear to be the case, so can these two fields both be made `private` or `protected`?